### DISCUSSION TITLE
Interaction Between FCU and Logitech Flight Panel (Cold Start Verses At Takeoff Runway Start

### DISCUSSION THREAD

**Discussion by FlyAwayIn2023** [flybywiresim/aircraft#8301] [Help] [answered]
I have noticed that if you are using the Logitech Control Panel (autopilot) the FBW A320 recognizes inputs (Alt/VS/Heading/Airspeed) when the flight has been started from a runway start (everything up and running).  These features do not seem to function if you have performed a "cold and dark" start from a gate.  SPAD.Next did not seem to effect these two start configurations.  This has not been a case with other aircraft.  Thanks guys for a really great product - I love it! 

MSFS Version [1.34.16.0]
A32NX  - Development Version e1bf51e

Latest version of Logitech Control Panel software.

To clarify - the aircraft in-game panels works great, the problem is between the Logitech external autopilot and how the aircraft responds in the two above startup scenarios.  

![image](https://github.com/flybywiresim/aircraft/assets/150606437/532095e8-a59b-4297-a381-30d8973fc2ba)

**Comment by alepouna** [comment, marked as answer]
You have mentioned spad.next, I assume you use that for your hardware to sim communication, I recommend you contact spad.next directly if you haven't already as it could be an issue on their end (or your configuration / preset used)
